index.es.md

```diff
@@ -0,0 +1,246 @@
+
+Es más eficiente y efectivo incorporar la accesibilidad desde el principio de los proyectos. De esta manera posteriormente no será necesario volver atrás y rehacer el trabajo.
+
+{::nomarkdown}
+{% include box.html type="start" h="3" title="Más información sobre cómo Hacer su Sitio Web Accesible" class="simple aside" %}
+{:/}
+
+-   Para tener una introducción a los requisitos de accesibilidad y estándares internacionales, acceda a [[Principios de Accesibilidad]](/fundamentals/accessibility-principles/).
+-   Para entender algunas de las barreras comunes de accesibilidad desde la perspectiva de la evaluación, acceda a [[Pruebas sencillas - Una Primera Revisión]](/test-evaluate/preliminary/).
+-   Para conocer algunas de las consideraciones básicas de diseño, edición y desarrollo para accesibilidad, acceda a [[Consejos Para Empezar]](/tips/).
+-   Cuando esté preparado para saber más sobre desarrollo y diseño, probablemente le serán útiles recursos como:
+    -   [Cómo cumplir las WCAG (Referencia Rápida)](http://www.w3.org/WAI/WCAG21/quickref/)
+    -   [Tutoriales de Accesibilidad Web](https://www.w3.org/WAI/tutorials/)
+-   Para la gestión de proyectos y consideraciones organizativas, acceda a [[Planificación y Gestión de la Accesibilidad Web]](/planning-and-managing/).<br>
+    Si necesita hacer correciones rápidas, acceda a [[Estrategias para Reparaciones Provisionales]](/planning/interim-repairs/).
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+	
+## Evaluar la Accesibilidad {#evaluate}
+
+Al desarrollar o rediseñar un sitio web, evalúe la accesibilidad desde el principio y durante todo el proceso de desarrollo para identificar los problemas de accesibilidad desde el inicio, cuando es más fácil resolverlos. Pasos sencillos, como cambiar la configuración del navegador, pueden ayudarle a evaluar algunos aspectos de la accesibilidad. Una evaluación exhaustiva para determinar si un sitio web cumple con todas las pautas de accesibilidad requiere más esfuerzo.
+
+Existen herramientas que ayudan en la evaluación. Sin embargo, ninguna herramienta por sí sola puede determinar si un sitio cumple con las pautas de accesibilidad. Se requiere una evaluación humana bien experimentada para determinar si un sitio es accesible.
+
+
+{::nomarkdown}
+{% include box.html type="start" h="3" title="Más información sobre cómo Evaluar Accesibilidad" class="simple aside" %}
+{:/}
+
+-   Los recursos para ayudar en la evaluación de accesibilidad se describen en [[Evaluar la Accesibilidad de Sitios Web]](/test-evaluate/).
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{% include excol.html type="start" id="examples" %}
+
+## Ejemplos
+
+{% include excol.html type="middle" %}
+
+### Alternativas textuales para imágenes
+
+![image of logo; HTML markup img alt='Iniciativa de Accesibilidad Web'](https://www.w3.org/WAI/intro/alt-logo.png){:.right}
```

Comment:
This is alternative text for [this image](https://www.w3.org/WAI/intro/alt-logo.png). For at least a while, we will not change the text in the image (we might sometime later when we get to make it SVG , but it's not a priority for now). Therefore, I think should translate:

"image of logo" and "markup"

and leave in English "Web Accessibility Initiative logo"

Reply:
The test "image of logo; HTML markup img alt='Iniciativa de Accesibilidad Web'" could be translated as "Imagen de logotipo; Marcado HTML img alt='Web Accessibility Initiative logo'".